tests/test_integration_service.py

```diff
@@ -1084,6 +1039,7 @@ def test_detail_list(self):
                     'it': True,
                     'leave': False,
                     'manager': False,
+                    'org_unit': False,
```

Comment:
This is a bit silly. 

Following my previous comment, I don't think we should have the same set of relation details available for both org units and employees.

Reply:
Eh, which comment would that be?

Reply:
A comment on commit `0a4cbac`. It shows up as 'outdated' because of your merge, I guess.